src/components/EditOrganization/index.tsx

```diff
@@ -172,24 +194,34 @@ export default function OrganizationView({ organization, reloadOrganizationData
             />
           </Grid>
           <Grid item xs={gridSize()} paddingBottom={theme.spacing(4)}>
-            <Select
-              label={strings.COUNTRY}
+            <Autocomplete
```

Comment:
wondering if we should create a dropdown component (like we do for select) that handles {label,value} semantics... avoid some repetition

Reply:
Yes, that would be useful

Reply:
ok, it was a suggestion..  
will do this as part of 2781